src/modules/endpoint/determineApiUrl.js

```diff
@@ -1,10 +1,10 @@
 // @flow
 
 import AppSettings from '../settings/AppSettings'
-import { baseUrl } from './constants'
+import buildConfig from '../app/constants/buildConfig'
 
 export default async () => {
   const appSettings = new AppSettings()
   const apiUrlOverride = await appSettings.loadApiUrlOverride()
-  return apiUrlOverride || baseUrl
+  return apiUrlOverride || buildConfig().cmsUrl
```

Comment:
I guess, this should depend on `__DEV__` like 
```suggestion
  return apiUrlOverride || (__DEV__ ? buildConfig().testCmsUrl : buildConfig().cmsUrl)
```

Reply:
No, it should depend on the build config.

Reply:
At least that is my opinion. Otherwise we would not need a integreat-test-cms build config.

Reply:
I'm fine with the current state. Doesn't really make a huge difference imo.